documentation/changes.md

```diff
@@ -0,0 +1,68 @@
+# Breaking Change Policy
```

Comment:
I think in general, customers ignore deprecation warnings, and only notice when things stop working, and thus the flow really ought to be:

1) Deprecation warnings for deprecated commands/features
<N months pass>
2) Defaults change, things are semi-removed, they stop working by default _but_ there is a `--activate-deprecated` flag that restores things.  This way customers notice, but they have a rollback path.
<N months pass>
3) Things are removed for realz.